projects/vdk-core/plugins/vdk-logging-json/src/vdk/plugin/logging_json/logging_json.py

```diff
@@ -19,14 +19,14 @@
 # this class serves to escape newline characters in the log message
 # as according to https://stackoverflow.com/questions/42068/how-do-i-handle-newlines-in-json
 # it is currently experimental and might be removed
-class RemoveNewlinesFormatter(logging.Formatter):
+class EscapeNewlinesAndQuotesFormatter(logging.Formatter):
```

Comment:
I do not see change in tests. 

Most changes but bug fix especially should be covered in tests - 
a) to make sure the fix is truly fixed (the test should fail before the fix
b) to make sure we do not regress again as bug tend to resurface 
c) make it easier to understand the problem and review

Reply:
Included example logs. As for the issue, a JSON parser will stop at the first double quote, as it sees it as the end of the message field, but the extra text afterwards causes a parsing error.

Will include a more extensive test as the formatter subclass was not being tested until now.